nixos/modules/installer/tools/nixos-install.sh

```diff
@@ -102,6 +102,7 @@ if [[ -z $system ]]; then
     outLink="$tmpdir/system"
     nix build --out-link "$outLink" --store "$mountPoint" "${extraBuildFlags[@]}" \
         --extra-substituters "$sub" \
+        --option experimental-features nix-command \
```

Comment:
We could just change this back to `nix-build` (making `nixos-install` consistent with `nixos-rebuild`).

Reply:
I can do that but I thought we wanted to dogfood the stuff.

Reply:
Dogfooding == using it yourself, not making other users use it :-)

Reply:
Do we revert e88f28965a7d76e83478d3ae6fcddc165b1c94f1?

Reply:
No, I think we only have to change `nix build` to `nix-build`. IIRC it also supports the `--store` argument which is what allows an install into a chroot.